doc/getting_started_linux.rst

```diff
@@ -160,14 +163,78 @@ To change the ephemeral port range::
 This should typically be done at system start up.
 
 
+File size and memory usage on Linux
+-----------------------------------
+The resulting file size of the sample application binary is heavily dependent
+on the compile time options, for example whether to include debug information::
+
+    p-net/build$ ls -l pn_dev
+    -rwxr-xr-x 1 jonas jonas 365984 sep  9 15:37 pn_dev
+
+    p-net/build$ file pn_dev
+    pn_dev: ELF 64-bit LSB shared object, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, for GNU/Linux 3.2.0, BuildID[sha1]=d2c45d5a17c1ad63db4f28c82cc04f8cfe5dcbfd, not stripped
+
+    p-net/build$ size pn_dev
+       text	   data	    bss	    dec	    hex	filename
+    337054	   1240	     24	 338318	  5298e	pn_dev
+
+See https://linux.die.net/man/1/size for information on how to use the command.
+Also the rt-kernel page in this documentation has some description on how to
+interpret the output.
+
+Add the ``-A`` flag to have more detailed information. If you build with debug
```

Comment:
I don't see that size -A adds anything interesting?

Reply:
Removed